benchmark/check.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# This script compiles all plugins and run all prm files in the subdirectories
+# of the benchmark folder
+
+if [ "$#" -ne 1 ]; then
+    echo "usage: $0 aspect-build-directory"
+    exit 1
+fi
+
+BUILD=`cd $1;pwd`
+
+if [[ ! -e $BUILD/AspectConfig.cmake || ! -e $BUILD/aspect  ]];
+then
+    echo "'$BUILD' doesn't look like a build directory"
+    exit 1
+fi
+
+
+# run aspect on all .prm files in the current folder or any subdirectory
+run_prm ()
+{
+    for prm in `find . -name "*prm"`;
+    do
+    if [ "`basename $prm`" = "parameters.prm" ];
+    then
+	continue;
+    fi
+    echo "Running '$prm' at `pwd` with '$BUILD' ..."
+    cp $prm tmp.prm~
```

Comment:
I think it would be nice to use a different name for this tmp file that doesn't indicate that it's an emacs backup file. Maybe just `tmp.prm`, or `$prm.instantaneous` or similar?